contracts/src/storage/CoreStorage.sol

```diff
@@ -14,7 +14,7 @@ library CoreStorage {
         // Agents
         mapping(bytes32 agentID => address) agents;
         // Reserve slot to prevent state collision
-        uint256 private __gap;
```

Comment:
@claravanstaden Seems like a bug introduced in https://github.com/Snowfork/snowbridge/pull/1439?

Reply:
Thanks! Fix here: https://github.com/Snowfork/snowbridge/pull/1465